demo/predict-taxi-trip-duration-nb/demo/README.md

```diff
@@ -48,5 +48,89 @@ python3 predict.py
 [[ 2.       40.774097 40.774097 40.774097 40.774097 40.774097 40.774097
   40.774097 40.774097  1.        1.      ]]
 ---------------predict trip_duration -------------
-859.3298781277192 s
+848.014745715936 s
+```
+To read more details about cluster mode, please refer [here](https://github.com/4paradigm/OpenMLDB/blob/main/docs/en/cluster.md)
+
+## Running the demo with standalone mode
+Start docker
+```bash
+docker run -it ghcr.io/4paradigm/openmldb:0.3.0 bash
+```
+Initilize environment
+```bash
+./init.sh standalone
 ```
+Create table and import the data to OpenMLDB. The recent data will be imported again in the real scene before deploying online service.
```

Comment:
remove "The recent data will be imported again in the real scene before deploying online service." here. We will explain this when deploying the online sql